<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLookConfig</class>
 <widget class="QDialog" name="DlgLookConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuración Apariencia</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Apariencia</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tamaño texto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Altura filas</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="rowHeightSpinBox">
        <property name="minimum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="textSizeSpinBox">
        <property name="minimum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QTableWidget" name="coloursTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Alta/Baja</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Alta</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Conectado</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Baja temporal</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Desconectado</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Baja administrativa</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Baja técnica</string>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>Baja indefinida</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>Baja definitiva, antena retirada</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="styleGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Estilos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="textFontButton">
        <property name="text">
         <string>Fuente texto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="textColorButton">
        <property name="text">
         <string>Color Texto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="backColorButton">
        <property name="text">
         <string>Color fondo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="textRedSpinBox">
        <property name="prefix">
         <string>Rojo </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="backRedSpinBox">
        <property name="prefix">
         <string>Rojo </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="textGreenSpinBox">
        <property name="prefix">
         <string>Verde </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="backGreenSpinBox">
        <property name="prefix">
         <string>Verde </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QSpinBox" name="textBlueSpinBox">
        <property name="prefix">
         <string>Azul </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="backBlueSpinBox">
        <property name="prefix">
         <string>Azul </string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="boldCheckBox">
        <property name="text">
         <string>Negrita</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="italicCheckBox">
        <property name="text">
         <string>Cursiva</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
